analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -638,7 +638,7 @@ def parse_convert_reports(input_dirs: List[str],
 
     number_of_reports = len(all_reports)
     if out_format == "codeclimate":
-        return codeclimate.convert(all_reports), number_of_reports
+        return codeclimate.convert(all_reports, severity_map), number_of_reports
```

Comment:
Please update the convert function here with the serverity map:
https://github.com/Ericsson/codechecker/blob/9511fd029f53180bfec3d127306f841ba4a1d427/web/client/codechecker_client/cmd_line_client.py#L1231

Reply:
fixed